lisa/tools/lsblk.py

```diff
@@ -155,57 +151,42 @@ def get_disks(self, force_run: bool = False) -> List[DiskInfo]:
         disks: List[DiskInfo] = []
 
         # parse output of lsblk
+        # -b print SIZE in bytes rather than in human readable format
+        # -J output in JSON format
+        # -o list of columns to output
         output = self.run(
-            "-b -P -o NAME,SIZE,TYPE,MOUNTPOINT,FSTYPE", sudo=True, force_run=force_run
+            "-b -J -o NAME,SIZE,TYPE,MOUNTPOINT,FSTYPE", sudo=True, force_run=force_run
```

Comment:
there is a regression for this change.
On some images like `openlogic centos 7_9-arm64 7.9.2023010302`, lsblk doesn't support -J parameter, will fix it tomorrow.

Reply:
Thank you, Lili! We can use -J by default, if it doesn't work, use previous table format.